lib/src/gainmapmath.cpp

```diff
@@ -289,18 +290,10 @@ float pqOetfLUT(float e) {
 
 Color pqOetfLUT(Color e) { return {{{pqOetfLUT(e.r), pqOetfLUT(e.g), pqOetfLUT(e.b)}}}; }
 
-// Derived from the inverse of the Reference PQ OETF.
-static const float kPqInvA = 128.0f, kPqInvB = 107.0f, kPqInvC = 2413.0f, kPqInvD = 2392.0f,
-                   kPqInvE = 6.2773946361f, kPqInvF = 0.0126833f;
-
+// From Khronos DataFormat Spec 1.3, Section 13.6.1 - PQ Inverse OETF
```

Comment:
Same comment. Please remove this comment since we have comment for `pqOetf`, though the formula is the same